leobot_firmware/STM32F4_LeoBot/.project

```diff
@@ -0,0 +1,74 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
What kind of project is this? How it could be opened and compiled? Some short README is needed :)

Reply:
It's Atolic True Studio project: https://atollic.com/truestudio/

Reply:
It would be nice to have it mentioned in README not in this comments. Since majority of people will not look on this PR :)

Reply:
Created file with these comments.